build/copy-files-for-packaging.cmd

```diff
@@ -0,0 +1,32 @@
+@echo ==========================
+@echo Copying source files for NuGet symbols package
+xcopy /ys src\* %TO_PACK_TARGET%\src\src\
+
+@echo ==========================
+@echo Copying signed files to the NuGet lib path
+set PORTABLE_LIB=portable-net45+win
+
+md %TO_PACK_TARGET%\lib\%PORTABLE_LIB%\
+copy /y  %SIGNED%\*.dll "%NUGET_TARGET%\lib\%PORTABLE_LIB%\"
+copy /y %TO_SIGN_TARGET%\*.pdb "%NUGET_TARGET%\lib\%PORTABLE_LIB%\"
+copy /y %TO_SIGN_TARGET%\*.xml "%NUGET_TARGET%\lib\%PORTABLE_LIB%\"
+
+call %COPY_TO_NUGET_LIBFOLDER% Desktop net45
```

Comment:
You're right - I deleted `%COPY_TO_NUGET_LIBFOLDER%` and didn't put it back. 

On the build, I am expecting it to fail because obviously there are no signed files to package up - that's next on my to-do list!